### DISCUSSION TITLE
怎样删除五笔下产生的自动造词

### DISCUSSION THREAD

**Discussion by SolidBlock-cn** [fcitx5-android/fcitx5-android#434] [Q&A | 问答]
有时候意外地错误制造了一些五笔自动造词的词汇。怎样从词库中删除这些词语？我长按候选结果没有反应。
我使用的是安卓版本。

**Comment by rocka** [comment]
目前虚拟键盘还没实现删词的功能。如果你可以连接蓝牙或者 USB 键盘，可以使用快捷键 Ctrl+7 删词。或者找到用户数据目录，删除用户词库文件 `data/table/<table_name>.user.dict`。
